src/python/gudhi/off_utils.pyx

```diff
@@ -25,21 +26,100 @@ __license__ = "MIT"
 cdef extern from "Off_reader_interface.h" namespace "Gudhi":
     vector[vector[double]] read_points_from_OFF_file(string off_file)
 
+def _get_next_line(file_desc, comment='#'):
+    """Return the next line that is not a comment.
+
+    :param file_desc: An open file in read mode.
+    :type file_desc: file
+    :param comment: The characters or list of characters used to indicate the start of a comment.
+    :type comment: string
+
+    :returns: The next line.
+    :rtype: string
+    """
+    while True:
+        # file_desc.readline() is preferred to next(file_desc), as the second option is not compatible with
+        # seek and tell methods
+        line = file_desc.readline()
+        if not line.startswith(comment):
```

Comment:
The doc of geomview says that blank lines are acceptable anywhere. Do we do the right thing if there is a blank line? Or do we need something like `l=line.lstrip()` `if len(l) != 0 and ...`?